packages/protocol/contracts/layer1/preconf/impl/PreconfWhitelist.sol

```diff
@@ -210,25 +220,27 @@ contract PreconfWhitelist is EssentialContract, IPreconfWhitelist {
         require(info.inactiveSince == 0, OperatorAlreadyRemoved());
         require(info.activeSince != 0, InvalidOperatorAddress());
 
+        address driver = info.driverAddress;
+
         uint8 _lastOperatorIndex = operatorCount - 1;
         if (_operatorChangeDelay == 0 && operators[_operator].index == _lastOperatorIndex) {
             // If delay is 0 and operator is the last one, remove directly
             delete operators[_operator];
             delete operatorMapping[_lastOperatorIndex];
             operatorCount = _lastOperatorIndex;
-            emit OperatorRemoved(_operator, block.timestamp);
+            emit OperatorRemoved(_operator, driver, block.timestamp);
         } else {
-            uint64 inactiveSince = epochStartTimestamp(_operatorChangeDelay);
+            uint32 inactiveSince = epochStartTimestamp(_operatorChangeDelay);
             operators[_operator].inactiveSince = inactiveSince;
             operators[_operator].activeSince = 0;
 
             havingPerfectOperators = false;
-            emit OperatorRemoved(_operator, inactiveSince);
+            emit OperatorRemoved(_operator, driver, inactiveSince);
         }
     }
 
     /// @dev The cost of this function is primarily linear with respect to operatorCount.
-    function _getOperatorForEpoch(uint64 _epochTimestamp) internal view returns (address) {
```

Comment:
can't see in the diff, will pull down this PR and ensure this function is still returning what is expected and leave a further comment! (on mobile)